crates/uv-python/src/version_files.rs

```diff
@@ -70,33 +78,38 @@ impl PythonVersionFile {
         working_directory: impl AsRef<Path>,
         options: &DiscoveryOptions<'_>,
     ) -> Result<Option<Self>, std::io::Error> {
-        let Some(path) = Self::find_nearest(&working_directory, options) else {
-            if let Some(stop_discovery_at) = options.stop_discovery_at {
-                if stop_discovery_at == working_directory.as_ref() {
-                    debug!(
-                        "No Python version file found in workspace: {}",
-                        working_directory.as_ref().display()
-                    );
+        let allow_local = !options.no_local;
+        let Some(path) = allow_local.then(|| {
+            // First, try to find a local version file.
+            let local = Self::find_nearest(&working_directory, options);
+            if local.is_none() {
+                // Log where we searched for the file, if not found
```

Comment:
These messages are just indented, we don't want to show them when `no_local` is enabled.

Reply:
What do you mean by just indented?